File: app/src/main/java/com/stratagile/qlink/ui/activity/main/contract/TestContract.java
package com.stratagile.qlink.ui.activity.main.contract;

import com.stratagile.qlink.ui.activity.base.BasePresenter;
import com.stratagile.qlink.ui.activity.base.BaseView;
/**
 * @author hzp
 * @Package The contract for TestActivity
 * @Description: $description
 * @date 2018/12/18 11:09:36
 */
public interface TestContract {
    interface View extends BaseView<TestContractPresenter> {
        /**
         *
         */
        void showProgressDialog();

        /**
         *
         */
        void closeProgressDialog();
    }

    interface TestContractPresenter extends BasePresenter {
//        /**
//         *
//         */
//        void getBusinessInfo(Map map);
    }
}

File: app/src/main/java/com/stratagile/qlink/ui/activity/test/contract/TestContract.kt
package com.stratagile.qlink.ui.activity.test.contract

import com.stratagile.qlink.ui.activity.base.BasePresenter
import com.stratagile.qlink.ui.activity.base.BaseView

/**
 * @author hzp
 * @Package The contract for TestFragment
 * @Description: $description
 * @date 2018/09/10 16:51:54
 */
interface TestContract {
    interface View : BaseView<TestContractPresenter> {
        /**
         *
         */
        fun showProgressDialog()

        /**
         *
         */
        fun closeProgressDialog()
    }

    interface TestContractPresenter : BasePresenter//        /**
    //         *
    //         */
    //        void getBusinessInfo(Map map);
}